README.md

```diff
@@ -1 +1,46 @@
-# react-capstone
\ No newline at end of file
+# Crypto App
+
+> This project was built to display a website about cryptocurrencies. In this website you can view 100 cyptos their name,rank and price on the home page the homepage can be filtered and you can search for your desired crypto among them. You can click on one of the cryptos and it will lead you to the details page of the crypto there you can see more information regarding the crypto that you have selected. All these datas are fetched from the API and are shown on the screen here.
+
+
+## Built With
+
+- Javascript
+- React, Redux
+- Webpack, Babel, React Testing Library
+
+## Live Demo
+https://sage-sprite-c442d7.netlify.app
+
+## Getting Started
+
+To start with this project. You have to run npm install. When you want to build the project run npm build and to start the project and run it on a browser run npm start. 
+
+### Prerequisites
+
+You just need a good computer and a code editor. 
+
+## Authors
+
+👤 ** Abdul Rahman Daudzai **
+
+- GitHub: [@wadaudzai](https://github.com/wadaudzai)
+- Twitter: [@wadaudzai](https://twitter.com/wadaudzai)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/wadaudzai)
+
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/wadaudzai/react-capstone/issues).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
```

Comment:
- [ ] Please know that the README contains an "Acknowledgements" section or similar that gives appropriate credit to [Nelson Sakwa](https://www.behance.net/sakwadesignstudio) the [author of the original design](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)), as required by the [Creative Commons License](https://creativecommons.org/licenses/).

Reply:
Changes are made according to what you instructed. Thank you !